ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -805,11 +805,11 @@
                 "\n        switch (periodType) {",
                 "\n            case PeriodType.BeginningOfPeriod :",
                 "\n                for (var i = 0; i < parentDiscountedValues.Length; i++)",
-                "\n                     ret[i] = -1d * (parentDiscountedValues[i] - parentNominalValues[i]) * (GetElementOrDefault(monthlyInterestFactor, i/12) - 1d );",
+                "\n                     ret[i] = -1d * (parentDiscountedValues[i] - parentNominalValues[i]) * (GetValidElement(monthlyInterestFactor, i/12) - 1d );",
```

Comment:
let's make use of "this"
=> monthlyInterestFactor.GetValidElement(i/12)

Reply:
Good idea, I applied the proposed syntax in every call of `GetValidElement`.